src/js/views/patients/worklist/action_views.js

```diff
@@ -74,9 +75,16 @@ const ActionItemView = View.extend({
   onClickPatient() {
     Radio.trigger('event-router', 'patient:dashboard', this.model.get('_patient'));
   },
-  onClickSelect() {
+  onClickSelect(param, domEvent) {
     const isSelected = this.state.isSelected(this.model);
+
+    if (domEvent.shiftKey && this.state.get('lastSelectedAction') && !isSelected) {
+      this.triggerMethod('click:shiftMultiSelect', this.model.id);
+      return;
+    }
+
     this.state.toggleSelected(this.model, !isSelected);
+    this.state.setLastSelectedAction(!isSelected ? this.model.id : null);
     this.render();
   },
```

Comment:
Intercepts the click event on an individual action in the list. If the `shiftKey` event property is `true`, it means the user was holding the shift key when they made the click.

If `shiftKey === true`, a last selection action `id` value exists in the state, and the user wasn’t un-selecting the action, trigger a `click:shiftMultiSelect` method that is passed to the `worklist_views.js` file to handle (where we can access the full list of actions in the list). 

Couple notes:

- `this` and the jQuery click event are passed to the function as parameters. I needed to pass both parameters in to get access to the jQuery event, but didn’t know what to call the first one. So just named it `param` for now.

- In this initial implementation, I only built the `shift`+`click` functionality for selecting items. And not for un-selecting items. So if the user is un-selecting an item, I don’t trigger the `click:shiftMultiSelect` method and I also clear the `lastSelectedAction` state value if that’s the case.